src/service/environmental_authority.js

```diff
@@ -208,6 +208,26 @@ module.exports = (eaPersistence, seService) => {
       return { total_area: eaArea[0].area };
     },
 
+    /**
+     * Get the information the current area distribution for each human footprint category in the
+     * given environmental authority
+     * @param {String} eaId environmental authority id
+     *
+     * @returns {Object[]} Array of areas by human footprint category with their respective
+     * percentage
+     */
+    getAreaByHFCategory: async (eaId) => {
+      let eaArea = await envAuth.getTotalArea(eaId);
+      eaArea = eaArea.total_area;
+      const values = await eaPersistence.findAreaByHFCategory(eaId);
+      return values.map(value => ({
+        ...value,
```

Comment:
There is no point in copying the original object since you are overwriting all attributes.

Reply:
And this applies for all services

Reply:
Done